src/components/safe-messages/MsgSigners/index.tsx

```diff
@@ -85,7 +92,7 @@ export const MsgSigners = ({
       {!hideConfirmations &&
         confirmations.map(({ owner }) => (
           <ListItem key={owner.value} sx={{ py: 0 }}>
-            <ListItemIcon>
+            <ListItemIcon sx={{ backgroundColor: ({ palette }) => backgroundColor ?? palette.background.paper }}>
```

Comment:
In my tests, we don't need the fallback and therefore the `palette` destructuring in these `ListItemIcon`s.